Make concrete sensor IDL tests use standalone IDL files

Honry opened the pull request:
Concrete sensor APIs: Accelerometer, Gyroscope,
 Magnetometer, Orientation Sensor

<!-- Reviewable:start -->

<!-- Reviewable:end -->

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20009)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/294802155)

alexshalamov commented on accelerometer/idlharness.https.html:
```diff
@@ -8,60 +8,32 @@
+    "/interfaces/generic-sensor.idl",
+    "/interfaces/accelerometer.idl",
+  ].map(fetchText)).then(doTest);
+}, "Test driver");
```
Should the name of the test reflect what is tested? Now it is "Test driver" for all sensors.

Honry replied:
This is a standalone test for `promise_test` itself, I can use more appropriate name.

alexshalamov commented on interfaces/orientation-sensor.idl:
```diff
@@ -0,0 +1,15 @@
+typedef (Float32Array or Float64Array or DOMMatrix) RotationMatrixType;
+
+[SecureContext, Exposed=Window]
+interface OrientationSensor : Sensor {
```
Does idlharness check that constructor is not defined to verify that interface is not constructible?

Honry replied:
Seems the answer is no, it is in TODO list at: https://github.com/w3c/web-platform-tests/blob/master/resources/idlharness.js#L1155

Honry committed:
Make concrete sensor IDL tests use standalone IDL files

 Concrete sensor APIs: Accelerometer, Gyroscope,
 Magnetometer, Orientation Sensor

Honry commented:
@alexshalamov, thanks!

Honry committed:
Improve test description
